enatega-multivendor-app/src/screens/Main/Main.js

```diff
@@ -449,15 +454,21 @@ function Main(props) {
                         )}
                       </View>
                     </View>
-                    <View>
-                      {orderLoading ? <TopBrandsLoadingUI /> : <TopBrands />}
-                    </View>
+                    {hasActiveOrders ? (
```

Comment:
whats the purpose of hasActiveOrders, if and else both have same body, will there be any difference ?

Reply:
yes the difference is in the styling (topBrandsMargin), we are giving conditional margin to the TopBrands so that they ca adjust themselves according to the height of the ActiveOrders's model.

Reply:
Then apply condition on styles instead of rendering whole component based on condition

Reply:
yeah, that's right. so should I apply its conditional styling in the view tag?